docs/articles/documentation/reference/test-api/testcontroller/dispatchEvent.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: docs
+title: t.dispatchEvent Method
+permalink: /documentation/reference/test-api/testcontroller/dispatchevent.html
+---
+# t.dispatchEvent Method
+
+```text
+t.dispatchEvent(target, eventName[, options]) → this | Promise<any>
```

Comment:
> TestCafe fires [DOM events](https://developer.mozilla.org/en-US/docs/Web/API/Event) to imitate user actions on the webpage. For example, when you call [t.click](./click.md), TestCafe raises `mousedown`, `mouseup` and `click` (and possibly other events if necessary) in succession.
>
> With `t.dispatchEvent` you can fire separate events on the webpage. This can be useful to imitate actions TestCafe does not support out-of-the-box.

Reply:
You change the sense:

Use the `t.dispatchEvnet` if you need to raise the single customized event or emulate actions what is not supported TestCafe now.

With t.dispatchEvent you can fire separate events on the webpage. This can be useful to imitate actions TestCafe does not support out-of-the-box.

Reply:
> Use t.dispatchEvent to raise customized events or to imitate actions TestCafe does not support out-of-the-box.